tool/gravity/cli/operation.go

```diff
@@ -182,6 +182,17 @@ func rollbackPlan(localEnv *localenv.LocalEnvironment, environ LocalEnvironmentF
 	default:
 		return trace.BadParameter(unsupportedRollbackWarning, op.TypeString())
 	}
+
+	statusList, err := collectAgentStatus(localEnv)
```

Comment:
This code block is basically identical to the one in the "resume", can we factor it out into a helper method?